scripts/generate.js

```diff
@@ -44,6 +53,43 @@ for (let component of components) {
   fs.writeFileSync(`${__dirname}/../build/${component.name}Icon.js`, fileContent);
 
   indexContent += `export ${component.name} from './${component.name}Icon';\n`;
+  
+  /*
+    Pushing the different formats of typescript definitions to their respective arrays.
+  */
+  defFileFillIns.declareIconClasses.push(`class ${component.name}Icon extends GenericMdiIcon {}`);
```

Comment:
Why not `extends MdiIcon {}` and `class MdiIcon extends __React.Component<IMdiIconProps, {}>{}` in line 84?